### DISCUSSION TITLE
click edge show modal form and line source to target show modal form

### DISCUSSION THREAD

**Discussion by Jafarammer** [xyflow/xyflow#3514] [Q&A]
### Please describe the feature that you want to propose

1. when the line from source to target displays a modal form, is that possible?
2.  when the edge line is clicked a modal form will appear, is that possible?

**Comment by moklick** [comment]
1. do you want to display a modal in the center of two connected nodes?
2. that is possible. You can use `onEdgeClick` for that.

**Reply by Jafarammer** [reply]
halo mr @moklick , If I use the pro version, can I request to make a custom work flow builder
